src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.test.js

```diff
@@ -26,7 +25,6 @@ describe('useTokensTopLskBalance hook', () => {
   it.skip('should fetch next set of data correctly', async () => {
     const { result, waitFor } = hookResult;
 
-    expect(result.current.isLoading).toBeTruthy();
```

Comment:
this test was passed, whats the reason?

Reply:
@soroushm Placeholder data was added to custom infinite query so it doesn't load at first, therefore `isLoading` will be false and then `isFetched` can be awaited for when the correct data is fetched

Reply:
its a bracking change we need to have isloading and showing loading state

Reply:
@soroushm We can use `isFetching` since that also changes when subsequent pages are being loaded